src/components/Home/Footer.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactElement } from 'react'
+import styles from './Footer.module.css'
+
+export default function Footer(): ReactElement {
+  const year = new Date().getFullYear()
+
+  return (
+    <footer className={styles.footer}>
+      <div className={styles.content}>
+        <div className={styles.copyright}>© {year} All Rights Reserved.</div>
```

Comment:
suggestion(non-blocking): We should use HTML encoding here (169 = ©) to support a wider range of fonts that might not support the character. Think also of custom fonts in IDEs used by developers.

Reply:
changed ✅